technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -364,11 +375,11 @@
     <group>
       <choose>
         <if type="article-journal article-magazine article-newspaper broadcast interview manuscript map patent personal_communication song speech thesis article webpage chapter paper-conference" match="any">
-          <label suffix="&#160;" variable="page" form="short"/>
+          <label suffix=" " variable="page" form="short"/>
```

Comment:
@jmiba, it looks like you replaced "&#160;" (non-breaking space) by regular spaces here and on lines 382 and 400 below. Is that intentional/desired?